Makefile

```diff
@@ -24,17 +26,28 @@ TAG ?=
 build:
 	@bazel $(BAZEL_STARTUP_ARGS) build $(BAZEL_BUILD_ARGS) //...
 
+# Build only envoy - fast
+build_envoy:
+	bazel $(BAZEL_STARTUP_ARGS) build $(BAZEL_BUILD_ARGS) //src/envoy/mixer:envoy
+
 clean:
 	@bazel clean
 
 test:
 	@bazel $(BAZEL_STARTUP_ARGS) test $(BAZEL_TEST_ARGS) //...
 
+test_envoy:
+	@bazel $(BAZEL_STARTUP_ARGS) test $(BAZEL_TEST_ARGS) //src/envoy/mixer/...
+
 check:
 	@script/check-license-headers
 	@script/check-style
 
 artifacts: build
 	@script/push-debian.sh -c opt -p $(ARTIFACTS_DIR)
 
+deb:
+	bazel build tools/deb:istio-proxy  ${BAZEL_BUILD_ARGS}
```

Comment:
I think you want ${BAZEL_BUILD_ARGS} before your targets

Reply:
Done.